packages/cashier/src/pages/on-ramp/on-ramp-provider-card/__tests__/on-ramp-provider-card.spec.tsx

```diff
@@ -13,10 +13,10 @@ describe('<OnRampProviderCard />', () => {
         is_dark_mode_on: false,
         is_mobile: false,
         provider: {
-            name: 'Changelly',
+            name: 'Banxa',
             icon: {
-                dark: 'IcCashierChangellyDark',
-                light: 'IcCashierChangellyLight',
+                dark: 'IcCashierBanxaDark',
+                light: 'IcCashierBanxaLight',
             },
             getDescription: jest.fn(
                 () =>
```

Comment:
@rostislav-deriv could we use description and payment icons from Banxa provider as well?